easybuild/easyconfigs/a/Arrow/Arrow-0.16.0-foss-2019b-Python-3.7.4.eb

```
@@ -0,0 +1,58 @@
+easyblock = 'CMakeMake'
+
+name = 'Arrow'
+version = '0.16.0'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'https://arrow.apache.org'
+description = """Apache Arrow (incl. PyArrow Python bindings)), a cross-language development platform
```

```suggestion
description = """Apache Arrow (incl. PyArrow Python bindings), a cross-language development platform
```

Removes extra bracket.